Build fail for tvOS

- [x] I have read [CONTRIBUTING](https://github.com/Quick/Nimble/blob/master/CONTRIBUTING.md) and have done my best to follow them.

### What did you do?

Build Nimbe-tvOS target

### What did you expect to happen?

Framework build.

### What actually happened instead?

Failure :

```
<unknown>:0: error: no such file or directory: '.../Sources/Lib/CwlPreconditionTesting/CwlPreconditionTesting/CwlCatchBadInstructionPosix.swift'
```

Log :


```
/Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble fix-nexted-bundles-issue*
xcodebuild -project Nimble.xcodeproj -target Nimble-tvOS
Build Preparation
Build task concurrency set to 5 via user default IDEBuildOperationMaxNumberOfConcurrentCompileTasks

=== BUILD TARGET Nimble-tvOS OF PROJECT Nimble WITH THE DEFAULT CONFIGURATION (Release) ===

Check dependencies

Write auxiliary files
write-file /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Nimble.hmap
/bin/mkdir -p /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/DerivedSources
write-file /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/DerivedSources/Nimble_vers.c
/bin/mkdir -p /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Objects-normal/arm64
write-file /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Objects-normal/arm64/Nimble-tvOS-OutputFileMap.json
write-file /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Nimble-project-headers.hmap
write-file /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/module.modulemap
write-file /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Objects-normal/arm64/Nimble.LinkFileList
write-file /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Nimble-all-target-headers.hmap
write-file /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Nimble-generated-files.hmap
write-file /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/unextended-module.modulemap
write-file /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Nimble-all-non-framework-target-headers.hmap
write-file /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/unextended-module-overlay.yaml
write-file /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/swift-overrides.hmap
write-file /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/all-product-headers.yaml
write-file /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Nimble-own-target-headers.hmap

Create product structure
/bin/mkdir -p /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Release-appletvos/Nimble.framework/Modules
/bin/mkdir -p /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Release-appletvos/Nimble.framework/Headers

ProcessInfoPlistFile build/Release-appletvos/Nimble.framework/Info.plist Sources/Nimble/Info.plist
    cd /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/AppleTVOS.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/Users/damiengavard/.rvm/gems/ruby-2.3.0/bin:/Users/damiengavard/.rvm/gems/ruby-2.3.0@global/bin:/Users/damiengavard/.rvm/rubies/ruby-2.3.0/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Users/damiengavard/Library/Android/sdk/tools:/Users/damiengavard/Library/Android/sdk/platform-tools:/Users/damiengavard/.rvm/bin"
    builtin-infoPlistUtility /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Info.plist -expandbuildsettings -format binary -platform appletvos -o /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Release-appletvos/Nimble.framework/Info.plist

CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler
    cd /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble
    export DEVELOPER_DIR=/Applications/Xcode.app/Contents/Developer
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/AppleTVOS.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/Users/damiengavard/.rvm/gems/ruby-2.3.0/bin:/Users/damiengavard/.rvm/gems/ruby-2.3.0@global/bin:/Users/damiengavard/.rvm/rubies/ruby-2.3.0/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Users/damiengavard/Library/Android/sdk/tools:/Users/damiengavard/Library/Android/sdk/platform-tools:/Users/damiengavard/.rvm/bin"
    export SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/AppleTVOS.platform/Developer/SDKs/AppleTVOS10.2.sdk
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc -incremental -module-name Nimble -O -whole-module-optimization -sdk /Applications/Xcode.app/Contents/Developer/Platforms/AppleTVOS.platform/Developer/SDKs/AppleTVOS10.2.sdk -target arm64-apple-tvos9.0 -g -Xfrontend -serialize-debugging-options -I /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Release-appletvos -F /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Release-appletvos -F /Applications/Xcode.app/Contents/Developer/Platforms/AppleTVOS.platform/Developer/Library/Frameworks -F /Applications/Xcode.app/Contents/Developer/Library/Frameworks -c -num-threads 5 /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/BeCloseTo.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Adapters/AssertionRecorder.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/MatcherProtocols.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Adapters/NimbleXCTestHandler.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/FailureMessage.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/BeLessThanOrEqual.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/AsyncMatcherWrapper.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/ThrowError.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/RaisesException.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/AllPass.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Utils/Errors.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/HaveCount.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/BeLogical.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Adapters/NMBExpectation.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Expression.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/BeAnInstanceOf.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/BeAKindOf.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/BeLessThan.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/BeGreaterThan.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/BeVoid.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/PostNotification.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/Contain.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/Equal.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Adapters/NMBObjCMatcher.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/DSL+Wait.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/BeGreaterThanOrEqualTo.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/MatchError.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Adapters/NimbleEnvironment.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Utils/Stringers.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Adapters/AdapterProtocols.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/BeginWith.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/BeIdenticalTo.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/MatcherFunc.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/BeEmpty.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Utils/Async.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/BeNil.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Adapters/AssertionDispatcher.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/ThrowAssertion.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/EndWith.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Utils/SourceLocation.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/DSL.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Expectation.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/SatisfyAnyOf.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/ContainElementSatisfying.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Utils/Functional.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Nimble/Matchers/Match.swift /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Lib/CwlPreconditionTesting/CwlPreconditionTesting/CwlCatchBadInstructionPosix.swift -output-file-map /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Objects-normal/arm64/Nimble-tvOS-OutputFileMap.json -parseable-output -serialize-diagnostics -emit-dependencies -emit-module -emit-module-path /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Objects-normal/arm64/Nimble.swiftmodule -Xcc -I/Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/swift-overrides.hmap -Xcc -iquote -Xcc /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Nimble-generated-files.hmap -Xcc -I/Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Nimble-own-target-headers.hmap -Xcc -I/Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Nimble-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/all-product-headers.yaml -Xcc -iquote -Xcc /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Nimble-project-headers.hmap -Xcc -I/Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Release-appletvos/include -Xcc -I/Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/DerivedSources/arm64 -Xcc -I/Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/DerivedSources -Xcc -DPRODUCT_NAME=Nimble/Nimble -emit-objc-header -emit-objc-header-path /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/Objects-normal/arm64/Nimble-Swift.h -import-underlying-module -Xcc -ivfsoverlay -Xcc /Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/build/Nimble.build/Release-appletvos/Nimble-tvOS.build/unextended-module-overlay.yaml -Xcc -working-directory/Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble
<unknown>:0: error: no such file or directory: '/Volumes/Develop/GitRepos/dagio/ReSwift-Router/Carthage/Checkouts/Nimble/Sources/Lib/CwlPreconditionTesting/CwlPreconditionTesting/CwlCatchBadInstructionPosix.swift'
Command /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc failed with exit code 1

** BUILD FAILED **


The following build commands failed:
	CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler
(1 failure)
```

### Environment

List the software versions you're using:

 - Quick: v1.1.0
 - Nimble: v6.1.0
 - Xcode Version: 8.3.2 (8E2002)
 - Swift Version: Xcode Default

Please also mention which package manager you used and its version. Delete the
other package managers in this list:

 - Cocoapods: N/A
 - Carthage: 0.20.1
 - Swift Package Manager: Swift 3.1.0 (swiftpm-12084)

### Project that demonstrates the issue

`xcodebuild -project Nimble.xcodeproj -target Nimble-tvOS`

--- a/Nimble.xcodeproj/project.pbxproj
+++ b/Nimble.xcodeproj/project.pbxproj
@@ -38,7 +38,7 @@
 		1F12BEA31E778FFA006952EC /* CwlPreconditionTesting.h in Headers */ = {isa = PBXBuildFile; fileRef = 1F12BE911E778F70006952EC /* CwlPreconditionTesting.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		1F12BEA41E77900A006952EC /* CwlBadInstructionException.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1F12BE8B1E778F70006952EC /* CwlBadInstructionException.swift */; };
 		1F12BEA51E77900A006952EC /* CwlCatchBadInstruction.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1F12BE8C1E778F70006952EC /* CwlCatchBadInstruction.swift */; };
-		1F12BEA61E779012006952EC /* CwlCatchBadInstructionPosix.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1F12BE8D1E778F70006952EC /* CwlCatchBadInstructionPosix.swift */; };
+		1F12BEA61E779012006952EC /* CwlCatchBadInstructionPOSIX.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1F12BE8D1E778F70006952EC /* CwlCatchBadInstructionPOSIX.swift */; };
 		1F12BEA71E779018006952EC /* CwlDarwinDefinitions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1F12BE8E1E778F70006952EC /* CwlDarwinDefinitions.swift */; };
 		1F12BEA81E77902A006952EC /* CwlPreconditionTesting.h in Headers */ = {isa = PBXBuildFile; fileRef = 1F12BE931E778F70006952EC /* CwlPreconditionTesting.h */; settings = {ATTRIBUTES = (Public, ); }; };
 		1F12BEDC1E7791B9006952EC /* CwlCatchException.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1F12BED71E7791B9006952EC /* CwlCatchException.swift */; };
@@ -480,7 +480,7 @@
 		1F12BE891E778F70006952EC /* mach_excServer.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = mach_excServer.h; sourceTree = "<group>"; };
 		1F12BE8B1E778F70006952EC /* CwlBadInstructionException.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = CwlBadInstructionException.swift; sourceTree = "<group>"; };
 		1F12BE8C1E778F70006952EC /* CwlCatchBadInstruction.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = CwlCatchBadInstruction.swift; sourceTree = "<group>"; };
-		1F12BE8D1E778F70006952EC /* CwlCatchBadInstructionPosix.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = CwlCatchBadInstructionPosix.swift; sourceTree = "<group>"; };
+		1F12BE8D1E778F70006952EC /* CwlCatchBadInstructionPOSIX.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = CwlCatchBadInstructionPOSIX.swift; sourceTree = "<group>"; };
 		1F12BE8E1E778F70006952EC /* CwlDarwinDefinitions.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = CwlDarwinDefinitions.swift; sourceTree = "<group>"; };
 		1F12BE911E778F70006952EC /* CwlPreconditionTesting.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = CwlPreconditionTesting.h; sourceTree = "<group>"; };
 		1F12BE931E778F70006952EC /* CwlPreconditionTesting.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = CwlPreconditionTesting.h; sourceTree = "<group>"; };
@@ -692,7 +692,7 @@
 			children = (
 				1F12BE8B1E778F70006952EC /* CwlBadInstructionException.swift */,
 				1F12BE8C1E778F70006952EC /* CwlCatchBadInstruction.swift */,
-				1F12BE8D1E778F70006952EC /* CwlCatchBadInstructionPosix.swift */,
+				1F12BE8D1E778F70006952EC /* CwlCatchBadInstructionPOSIX.swift */,
 				1F12BE8E1E778F70006952EC /* CwlDarwinDefinitions.swift */,
 				1F12BE901E778F70006952EC /* Mach */,
 				1F12BE921E778F70006952EC /* Posix */,
@@ -1489,7 +1489,7 @@
 				7B13BA0C1DD361D300C9098C /* ContainElementSatisfying.swift in Sources */,
 				1F5DF18B1BDCA0F500C3A531 /* Functional.swift in Sources */,
 				1F5DF1871BDCA0F500C3A531 /* Match.swift in Sources */,
-				1F12BEA61E779012006952EC /* CwlCatchBadInstructionPosix.swift in Sources */,
+				1F12BEA61E779012006952EC /* CwlCatchBadInstructionPOSIX.swift in Sources */,
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};

